project_report/tex/requirements.tex

```diff
@@ -9,7 +9,16 @@ \subsubsection{Functional \& Non-Functional Requirements}
 \subsubsection{The Physical Setup (The Brewery Machine)}
 
 \subsubsection{The Simulator}
-
+The group are going to use the simulator software to test their software in the 
+Development cycle. 
+The group need the simulation software to quickly make prototypes and see if it
+works probably and have no regressions since the last version before they pushed
+it to the production machine.
+The simulator is never going to beat the real thing because there is only
+so much randomness and correctness you can get of a simulator, but it is still
+very important to have the simulator, for making prototypes and make multiple
+tests for it, so they don't make any regressions in the production system. 
+The production system Is the beer machine.
```

Comment:
don't think  line 21 is necessary  maybe line 20:  ...the production system (the beer machine) instead